WoWPro_Leveling/Wrath/Horde/WOTLK_Borean_Tundra.lua

```diff
@@ -280,7 +269,6 @@ T A Courageous Strike |QID|11641|M|76.0,37.3|N|To Durm Icehide.|
 T Words of Power |QID|11640|M|75.9,37.2|N|To Chieftain Wintergale.|
 A Breaking Through |QID|11898|N|From Chieftain Wintergale.|M|75.8,37.2|
 R Go to|QID|11898|M|86.5,28.5|CC|N|the teleporter at the waypoint.|
```

Comment:
ACTIVE?

Reply:
Leaving this open cause im not sure which you are referring to.

Reply:
`R` steps should use ACTIVE or AVALIBLE and almost never QID.